rmc/utils/constraint.py

```diff
@@ -509,7 +520,15 @@ def create_constraint_prep_ht(
     :param overwrite: Whether to overwrite Table. Default is True.
     :return: None; writes Table to path.
     """
-    ht = filtered_context.ht()
+    # NOTE: Observed counts upstream now includes variants with AF <= 0.001 instead of AF < 0.001.
+    # ht = filtered_context.ht()
+    ht = get_per_variant_expected_dataset(
+        directory_post_fix="an_coverage_corrected", path_post_fix="coverage_corrected"
+    ).ht()
+    # Annotate obs and exp counts as the zero index (pop global) since they are arrays
```

Comment:
```suggestion
    # Obs and exp counts are arrays to account for different frequency strata
    # Use zeroth index (gnomAD-wide ["global"] frequencies)
```
maybe this (0) should also be an argument? something like
```
variant_idx: int = 0,
```

Reply:
Can be but I don't think this index will change since 0 always represented global intuitively?